### PR TITLE
[FEAT] ember-3x-codemods for version 3.1

### PR DESCRIPTION
**Pull request by rajasegar**
#21 
Use notifyPropertyChange instead of propertyWillChange and propertyDidChange
[Deprecations Added in 3.1](https://deprecations.emberjs.com/v3.x/#toc_use-notifypropertychange-instead-of-propertywillchange-and-propertydidchange)

### Input
```js
Ember.propertyWillChange(object, 'someProperty');
doStuff(object);
Ember.propertyDidChange(object, 'someProperty');

object.propertyWillChange('someProperty');
doStuff(object);
object.propertyDidChange('someProperty');
```

## Output
```js
doStuff(object);
Ember.notifyPropertyChange(object, 'someProperty');

doStuff(object);
object.notifyPropertyChange('someProperty');
```

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -32,7 +32,10 @@
     },
     "projectOptions": ["app", "addon"],
     "nodeVersion": "4.0.0",
-    "commands": ["github:rondale-sc/es5-getter-ember-codemod es5-getter-ember-codemod app/**/*.js"]
+    "commands": [
+      " github:rondale-sc/es5-getter-ember-codemod es5-getter-ember-codemod app/**/*.js",
```
Comment (kellyselden) [inline manifest.json]: this is in the es5-getter-ember-codemod block, you should make a new block.

Comment (rajasegar) [reply manifest.json]: New block added for ember-3x-codemods

File: manifest.json [hunk 2]
```diff
@@ -32,7 +32,19 @@
...
+    "commands": [
+      " github:rondale-sc/es5-getter-ember-codemod es5-getter-ember-codemod app/**/*.js"
+    ]
+  },
+  "notify-property-change": {
+    "versions": {
+      "ember-source": "3.1.0-beta.1"
+    },
+    "projectOptions": ["app", "addon"],
+    "nodeVersion": "4.0.0",
```
Comment (kellyselden) [inline manifest.json]: Are you sure ember-3x-codemods works in node 4? I don't see anything in that project that says which node versions are supported.

File: manifest.json [hunk 3]
```diff
@@ -32,7 +32,19 @@
     },
     "projectOptions": ["app", "addon"],
     "nodeVersion": "4.0.0",
-    "commands": ["github:rondale-sc/es5-getter-ember-codemod es5-getter-ember-codemod app/**/*.js"]
+    "commands": [
+      " github:rondale-sc/es5-getter-ember-codemod es5-getter-ember-codemod app/**/*.js"
```
Comment (kellyselden) [inline manifest.json]: Can you revert these changes from es5-getter-ember-codemod please?

Comment (rajasegar) [reply manifest.json]: Done

File: manifest.json (hunk 2 above)
Comment (rajasegar) [reply manifest.json]: Since it is using codemod-cli  0.2.10, which requires node 6, will update the same
https://github.com/rwjblue/codemod-cli/blob/v0.2.10/package.json

**Comment by kellyselden** [comment]
All good to go. Thanks again!

**Comment by rajasegar** [comment]
Thanks @kellyselden
